src/libraries/System.Data.Common/src/System/Data/DataView.cs

```diff
@@ -1134,10 +1141,16 @@ void IBindingListView.RemoveFilter()
         string IBindingListView.Filter
         {
             get { return RowFilter; }
-            set { RowFilter = value; }
+            set { SetRowFilter(value); }
         }
 #nullable enable
 
+        // This method is only needed so that ILLink suppression can be added
```

Comment:
I don't get this comment. What ILLink suppression?

Reply:
I meant suppression in the XML file. I wasn't able to suppress that initially the message was showing signature as `System.Data.DataView.em.ComponentModel.IBindingListView.set_Filter.set`.

I've tried couple more combination and this seems this have worked:

```
M:System.Data.DataView.System.ComponentModel.IBindingListView.set_Filter(System.String)
```

so will remove this method.

Reply:
When in doubt on the signature of the method in the XML file, put this change locally and run the linker:

https://github.com/dotnet/runtime/pull/44264/files

It will generate the XML file with the suppressions in it. You can copy the contents out of the generated file.